build/device/secure_ext.py

```diff
@@ -3,6 +3,8 @@
 
 MAGIK_CODE = [0x32, 0x30, 0x30, 0x36]
 MAGIK_POS = 0x44F
+# Disable Script
+sys.exit(0)

```

Comment:
I don't understand...

Reply:
We no longer use this type of protection because there is no longer an external/internal part but only bootloader and slots

Reply:
Yes, but why are you not removing it from the Makefile ?